posthog/api/feature_flag.py

```diff
@@ -200,6 +201,8 @@ def create(self, validated_data: Dict, *args: Any, **kwargs: Any) -> FeatureFlag
 
         report_user_action(request.user, "feature flag created", instance.get_analytics_metadata())
 
+        set_feature_flags_for_team_in_cache(self.context["team_id"])
```

Comment:
Should we have this on update too?

Reply:
whoops, nice catch thanks!

Reply:
I'll add some tests for this, cache should be in sync with all updates 👀

Reply:
All done!